### ISSUE TITLE
[WINDOW] ReadValue is called only ONCE

### ISSUE THREAD

**Issue by SandeepMalviyaSM** [boskokg/flutter_blue_plus#281]
Read Value is not working properly. Once you call the function, when the command is not passed to BLE Device.

**Comment by chipweinberger** [comment]
please try 1.7.6+, reopen a new issue if you still have problems.
